www/templates/default/html/manager/Calendar.tpl.php

```diff
@@ -0,0 +1,113 @@
+<?php
+    $events = $context->getCategorizedEvents();
+    $calendars = $context->getCalendars();
+?>
+<section class="wdn-grid-set">
+	<div class="wdn-col-one-fourth">
+		<h3>My Calendars</h3>
+        <?php foreach ($calendars as $calendar) { ?>
+            <?php if ($calendar->shortname == $context->calendar->shortname) { ?>
+                <div style="background: #CCCCCC;"><?= $calendar->name ?></div>
+            <?php } else { ?>
+                <a href="<?= $calendar->manageHref() ?>"><?= $calendar->name ?></a><br>
+            <?php } ?>
+        <?php } ?>
+        <a href="#" class="wdn-button wdn-button-brand">+ New Calendar</a>
+	</div>
+	<div class="wdn-col-three-fourths">
+        <ul class="wdn_tabs">
+            <li><a href="#pending">Pending (<?= count($events['pending']); ?>)</a></li>
+            <li><a href="#posted">Posted (<?= count($events['posted']); ?>)</a></li>
+            <li><a href="#archived">Archived (<?= count($events['archived']); ?>)</a></li>
+        </ul>
+        <div class="wdn_tabs_content">
+            <div id="pending">
+                <div id="pending-toolbar">
+                    <?= $context->calendar->name ?> <a href="#" class="wdn-button wdn-button-brand">+ New Event</a>
+                </div><br>
+                <div class="wdn-grid-set">
+                    <div class="wdn-col-two-thirds">
+                        <?php if (count($events['pending']) == 0) { ?>
```

Comment:
Lately, we have been trying to use the alternative syntax for control structures in templates because they are a little easier to follow.

http://php.net/manual/en/control-structures.alternative-syntax.php